File: javascript/fizzbuzz.js

var fizzBuzz = function(n) {
  if (n%15 === 0) {
    console.log("FizzBuzz");
  }
  else if (n%5 === 0) {
    console.log("Fizz");
  }
  else if (n%3 === 0) {
    console.log("Buzz");
  }
  else{
    console.log(n)
  }
};

fizzBuzz(3)
fizzBuzz(6)
fizzBuzz(9)
fizzBuzz(20)
fizzBuzz(15)
fizzBuzz(20)
fizzBuzz(50)
fizzBuzz(30)
fizzBuzz(23)

for (var number = 0; number <= 100; number ++) {
  fizzBuzz(number)
};

File: ruby/fizzbuzz.rb

def fizzbuzz(n)
  if n%15 == 0
    puts "FizzBuzz"
  elsif n%5 == 0
    puts "Buzz"
  elsif n % 3 == 0
    puts "Fizz"
  else
    puts n
  end
end

100.times do |x|
  fizzbuzz(x+1)
end

fizzbuzz(3)
fizzbuzz(6)
fizzbuzz(9)
fizzbuzz(10)
fizzbuzz(15)
fizzbuzz(30)
fizzbuzz(25)
fizzbuzz(35)
fizzbuzz(100)
